src/Configuration/Connections/MasterSlaveConnection.php

```diff
@@ -2,7 +2,7 @@
 
 namespace LaravelDoctrine\ORM\Configuration\Connections;
 
-use Doctrine\DBAL\Connections\MasterSlaveConnection as MasterSlaveDoctrineWrapper;
+use Doctrine\DBAL\Connections\PrimaryReadReplicaConnection as MasterSlaveDoctrineWrapper;
```

Comment:
How challenging would it be to change the name of the alias as well? If I'm understanding the change that Doctrine underwent, `PrimaryReadReplicaConnection` is intending to replace problematic references in programming to slavery. Obviously this isn't something that would be a functional change, but would be in line with many movements that the software field has been addressing with legacy problematic naming conventions.